reference-architecture/osp-cli/docker-storage-setup

```diff
@@ -0,0 +1,2 @@
+DEVS=/dev/vdb
+VG=docker-vg
```

Comment:
You s houldn't need make-mime, you should be able to do it through cloud-config using write_files (example here: https://github.com/openshift/openshift-ansible/blob/master/playbooks/aws/openshift-cluster/templates/user_data.j2)